### ISSUE TITLE
Old rules that were copy and pasted need to be reworded/deprecated. 

### ISSUE THREAD

**Issue by insyri** [insyri/strafes.net-moderation-document-draft#10]
Some things were copy and pasted from the original document. Namely the [Bounces and Pocketing](https://github.com/insyri/strafes.net-moderation-document-draft/blob/main/rules/gameplay-sins.md#bouncingbounces--long-pocketing) are about the same, the only difference being the name and the intention of height application, and if they are not about the same thing, it is obscure text and needs to be reworded. Same thing with [Edgeboosting](https://github.com/insyri/strafes.net-moderation-document-draft/blob/main/rules/gameplay-sins.md#edgeboosting) with the surrounding `??`. And again with the map examples in the [no title rules](https://github.com/insyri/strafes.net-moderation-document-draft/blob/main/rules/gameplay-sins.md#no-title-rules) section.

TL;DR: confusion on old text, needs revising.

**Comment by insyri** [comment]
https://github.com/insyri/strafes.net-moderation-document-draft/commit/415245ee504965f85420eca5e2db1176702fb8af
